packages/bpk-docs/src/pages/GettingStartedPage/GettingStartedPage.js

```diff
@@ -31,6 +31,8 @@ const nodeSassUrl = 'https://github.com/sass/node-sass';
 const sassLoaderUrl = 'https://github.com/jtangelder/sass-loader';
 const sassLoaderTildeUrl = 'https://github.com/jtangelder/sass-loader#imports';
 const nodeSassTildeImporterUrl = 'https://github.com/matthewdavidson/node-sass-tilde-importer';
+const iOSContributingLink = 'http://git.prod.skyscanner.local/apps-tribe/skyscanner-ios-app/blob/develop/CONTRIBUTE.md';
```

Comment:
we cant really point to internal links here can we?

Reply:
Dunno honestly. It won't resolve anyway so